### PR TITLE
feat: created a simple website as a frontend for the application, doc…

### LINKED ISSUE #29: 💡 [REQUEST] - Add a simple website to show off the app and the team.
### Reference Issues

_No response_

### Summary

We should have a simple website that gets shown off at the root of the api. This would be good for showing off the team etc.

Currently the root just shows a 404 page which isn't ideal.

### Basic Example

N/A

### Drawbacks

Would require upkeep, should be hostable from cloudflare pages.

### Unresolved questions

_No response_

### PR DESCRIPTION
**Pull request by JosiahBull**
## Todo

- [x] Update Dockerfiles
- [x] Check config
- [x] Write Website
- [x] Populate user blurbs etc

## Motivation

We should have a simple website that gets shown off at the root of the api. This would be good for showing off the team etc.

Currently the root just shows a 404 page which isn't ideal.

## Summary of changes

- [x] Website implemented, largely copied over from a previous implementation of mine.

## Attached GitHub issue

Closes #29

### Documentation
- [x] Updated the readme documents (`README.md` etc.)
- [x] New functions and methods have additional comments added to document their usage

### Local Build
- [x] Ran all pre-commit hooks `pre-commit run --all-files`
- [x] Extension has been tested to build correctly `cd extension && yarn && yarn build`
- [x] Backend has been tested to build correctly `cd backend && docker-compose build`
- [x] Extension test suite has been run locally `cd extension && yarn && yarn test`
- [x] Backend test suite has been run locally `cd backend && python3 -m unittest`

### GitHub
- [x] Target branch has been set correctly
- [x] PR has been rebased onto target branch
- [x] PR has been assigned an owner
- [x] Author has performed a self-review of the code
- [x] Removed the WIP message from the top of this PR

### CODE REVIEW

**Comment by JosiahBull** [comment]
Haven't got profile cards for Leonard or Lang, feel free to commit them before it's merged, or add them in a comment on this PR and I'll add them in before it's merged.

Don't merge this PR until that's been done.

**Review by hmin697 — APPROVED** [review]
nice!! incredible work

**Review by lche262 — APPROVED** [review]
Lgtm! Thanks for your work Jo!

**Pull request merged**
